### PR TITLE
Sudo

### PR DESCRIPTION
**Pull request by MichaelGeng**

### CODE REVIEW

**Comment by themoh** [comment]
Move all of the text you have under "" and include a shorter description above. See [cd](https://raw.githubusercontent.com/c4cs/c4cs.github.io/master/_commands/basics/cd.md) for an example

**Comment by MichaelGeng** [comment]
Okay I changed it as you asked. How does it look now?

**Comment by mterwill** [comment]
This doesn't show up on the reference page now... I think you moved your ``<!--more-->`` too high.

**Comment by mterwill** [comment]
This also looks wrong – bad markdown syntax or duplicate lines?

screen shot 2016-03-17 at 5 34 33 pm

**Comment by MichaelGeng** [comment]
Oh that was some weird formatting. It looks like I fixed it now?

File: _commands/basics/sudo.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+~~~ bash
+$ sudo apt-get install wbritish
+~~~
+
+By having sudo in the command we ran the command with elevated privileges and were able to install the british-english word list.
+
+### Useful Options / Examples
+There's some good commands you can run with sudo!
+
+sudo-b will run the command in the background.Useful to avoid long list of output as it runs
```
Comment (mterwill) [inline _commands/basics/sudo.md]: Use [Markdown code syntax](https://daringfireball.net/projects/markdown/syntax#code) for these examples. Also they're options, so there should be a space in between the command and the option.

**Comment by MichaelGeng** [comment]
Added markdown syntax and spaces

**Comment by mterwill** [comment]
You didn't actually use the markdown syntax for inline code on ``sudo -s``, etc. Surround code in backticks(`).

File: _commands/basics/sudo.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+---
+
+sudo
+---
+
+<!--more-->
+
+`sudo` (super user do) is a command that allows a permitted user to execute a command as a superuser or another user (as specified in the sudoers file). In other words, it runs the command with elevated privileges. 
+
```
Comment (mterwill) [inline _commands/basics/sudo.md]: Move `<!--more-->` here.

File: _commands/basics/sudo.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+sudo -b will run the command in the background.Useful to avoid long list of output as it runs
+
+sudo -s will run the shell specificed with elevated privlages, giving you the #prompt
+
+sudo -h will cause sudo to print a help message and exit
+
+sudo -l will list the allowed (and forbidden) commands for the invoking user on the current host
+
+There are many, many more which can be found online!
```
Comment (mterwill) [inline _commands/basics/sudo.md]: Or in the man pages!

**Comment by MichaelGeng** [comment]
Okay I added the markdown syntax around the commands and moved more around to make it actually appear on the site. How is it now?

**Pull request merged**